lcm/dbg.h

```diff
@@ -151,63 +118,62 @@ static void dbg_init()
         char *name;
 
         strncpy(env, dbg_env, sizeof(env));
-        for (name = strtok(env,","); name; name = strtok(NULL, ",")) {
+        for (name = strtok(env, ","); name; name = strtok(NULL, ",")) {
             int cancel;
             dbg_mode_t *mode;
 
             if (*name == '-') {
                 cancel = 1;
                 name++;
-            }
-            else
+            } else
                 cancel = 0;
 
             for (mode = dbg_nametab; mode->d_name != NULL; mode++)
-                if (strcmp(name, mode->d_name) == 0)
-                    break;
+                if (strcmp(name, mode->d_name) == 0) break;
```

Comment:
This is also a change vs. the current style.

Reply:
Let me see if there is a rule for this tonight.